src/main/menu.ts

```diff
@@ -157,6 +157,14 @@ export default class MenuBuilder {
         },
         enabled: false,
       },
+      {
+        id: 'export',
+        label: 'Export Project',
+        accelerator: 'CommandOrControl+Shift+E',
+        click: () => {
+          this.mainWindow.webContents.send('initiate-export-project');
```

Comment:
Good 👍🏻  but this won't work on windows or linux, check the pattern used for the save / save as and undo / redo

Reply:
Tested on windows and it works

Reply:
Ah my bad, you're right this is piggy backing on the existing buildFileOptions()